Applications/ResNet/res/resnet18.ini

```diff
@@ -154,12 +154,20 @@ kernel_size = 3,3
 bias_initializer = zeros
 weight_initializer = xavier_uniform
 filters=128
-stride =2,2
+stride =1,1
 padding=1,1
 
+[B2_pooling2d_1]
```

Comment:
How about instead of modifying the original resnet18.ini, we keep a resnet18-pooling.ini, so that resnet18.ini can be tested once @zhoonit resolves the multi-stride?